### PR TITLE
[Review] Request from 'aduffeck' @ 'SUSE/machinery/review_150112_extract_provisioning_code'

### PR DESCRIPTION
**Pull request by aduffeck**
Please review the following changes:
  * 9cc8272 Extract subject image provisioning code into an external script

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
( Success: Merged 9cc82725ec0d22c61b0317b4a2ce0894e60ba7ee into f8e4340656da679098a54e28f5c6ee35bca93e48 )

File: spec/definitions/provision.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# unmanaged-files
```
Comment (tgoettlicher) [inline spec/definitions/provision.sh]: `set -e` or `trap 'exit' ERR` could improve the robustness of this script.

**Comment by cornelius** [comment]
Looks good from my point of view

File: spec/definitions/provision.sh (hunk 1 above)
Comment (aduffeck) [inline spec/definitions/provision.sh]: You may be right, but I'd defer this change for now. The script was only moved here and it's working fine in its current form.

**Pull request merged**